web/package.json

```diff
@@ -13,13 +13,13 @@
     "svelte": "^3.44.0",
     "svelte-check": "^2.2.6",
     "svelte-toasts": "^1.1.2",
+    "svelte-typewriter": "^3.0.0-alpha.11",
     "tailwindcss": "^3.0.24",
     "typescript": "~4.6.2"
   },
   "type": "module",
-  "dependencies": {},
   "scripts": {
-    "dev": "svelte-kit --host 0.0.0.0 dev",
+    "dev": "svelte-kit dev",
```

Comment:
Any reason to remove this?

Reply:
This was essentially a typo, it wasn't supposed to be included in the commit. I'll submit another to remove it from the pull request.